mcd/mcd-seth/mcd-seth.md

```diff
@@ -125,192 +167,273 @@ In order to better understand the MCD contracts, the following provides a brief
 
-We can check the results with the contract function: `gem(bytes32 ilk,address urn)(uint256)` with.
+We can check the results with the contract function: `gem(bytes32 ilk, address urn)(uint256)` with:
 
-`seth --from-wei $(seth call $MCD_VAT 'gem(bytes32,address)(uint256)' $ilk $urn) eth`
+```bash
+seth call $MCD_VAT 'gem(bytes32, address)(uint256)' $ilk $urn | seth --from-wei
+```
 
 The output should look like this:
 
-`150.000000000000000000`
+`5.000000000000000000`
 
-An optional, but recommended step is to invoke `jug.drip(ilk)` to make we are not paying undue stability fees. For more detail, please read the guide [Intro to the Rate mechanism](https://github.com/makerdao/developerguides/blob/master/mcd/intro-rate-mechanism/intro-rate-mechanism.md)
+An optional, but recommended step is to invoke `jug.drip(ilk)` to make we are not paying undue stability fees. For more detail, please read the guide [Intro to the Rate mechanism](../intro-rate-mechanism/intro-rate-mechanism.md).
 
-`seth send $MCD_JUG 'drip(bytes32)' $ilk`
+```bash
+seth send $MCD_JUG 'drip(bytes32)' $ilk
+```
 
 The next step is adding the collateral into an urn. This is done through the `CDP Manager` contract.
-The function is called `frob(uint256,uint256,uint256)`, which receives couple of parameters:
 
-- `uint256 cdp` - the `cdpId`
-- `int256 dink` - delta ink (collateral)
-- `int256 dart` - delta art (Dai).
+The function is called `frob(uint256, uint256, uint256)`, which receives the following parameters:
+
+- `uint256 cdp`: the `cdpId`
+- `int256 dink`: delta ink (collateral)
+- `int256 dart`: delta art (Dai).
 
-If the `frob` operation is successful, it will adjust the corresponding data in the protected `vat` module. When adding collateral to an `urn`, `dink` needs to be the (positive) amount we want to add and `dart` needs to be the (positive) amount of DAI we want to draw. Let’s add our 150 BAT to the urn, and draw 20 DAI ensuring that the position is overcollateralized.
+If the `frob` operation is successful, it will adjust the corresponding data in the protected `vat` module. When adding collateral to an `urn`, `dink` needs to be the (positive) amount we want to add and `dart` needs to be the (positive) amount of DAI we want to draw. 
+
+Let’s add our 5 WBTC to the urn, and draw 15000 DAI ensuring that the position is overcollateralized.
+We already set up `cdp` before, so we only need to set up `dink` (WBTC deposit) and `dart` (DAI to be drawn):
+
+**ℹ️ NOTICE:** The `vat` uses an internal `dai` representation called “normalized art” that is useful to calculate accrued stability fees. To convert the Dai amount to normalized art, we have to divide it by the current ilk `rate`.
+
+Inside the `vat`, different parameters have different decimal precisions:
+
+- `dai`: 45 decimals (`rad`)
+- `rate`: 27 decimals (`ray`)
+- `dink`/`dart`: 18 decimals (`wad`).
```

Comment:
```suggestion
- `dink`: 18 decimals (`wad`)
- `dart`: 18 decimals (`wad`)
- `rate`: 27 decimals (`ray`)
- `dai`: 45 decimals (`rad`)
```

Reply:
Not sure if this should be sorted by the number of decimals. I made it match the usage order in the article.